examples/operations/QueueScript.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>Consider changing the title to "Import required libraries"</p><p>then the text can be:</p><p>"In the cell below, note we import <code>ATQueue</code>, and the code to import <code>MTQueue</code> is intentionally commented out. &nbsp;In both cases, the module is imported as <code>Queue</code> so the code in the cells below remains functional regardless of the chosen queue.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>